diagnostic.md

```diff
@@ -35,7 +39,9 @@ Record your responses inside the fenced code blocks below each question.
     ```
 
     ```md
-    <!-- your response here -->
+    1) in the first, product is nested within products, so the router will look within the products directory
+      to find the product directory, and will look there for associated files. in the second, the router will just look for the product directory in the first level from /app
+    2) the first is limited to a nested view-state, while the second could stand alone as an independent view-state
```

Comment:
You are right, nested route definitions, do require a nested file structure.  So the Route and Template files for `'product'` are expected to be found at `app/products/product`.

One of the biggest distinctions is where Ember will render the content.  
The nested `'products'/'product'` instructs Ember to render
`app/products/product/template.hbs` in the `outlet` of the parent `app/products/template.hbs`.  
The not-nested `'product'` instructs Ember to render `app/product/template.hbs` in the context of the application template.  

These two route definitions are identical  URL paths.